mh/equipment_utils.py

```diff
@@ -0,0 +1,1638 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Equipment utils module.
+
+    Monster Hunter 3 Server Project
+    Copyright (C) 2023  Ze SpyRo
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+import struct
+
+
+EMPTY_ARENA_EQUIPMENT_SLOT = struct.pack('>L', 0xFFFFFFFF)
+EMPTY_ARENA_ITEM_SLOT = struct.pack('>L', 0x00000000)
```

Comment:
`struct.pack('>I',` should be used instead.

Reply:
Fixed and pushed.